.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
- [ ] [REQUIRED] Good job so far! On my side few `specs` failed when they run, please fix them if your side everything is ok please show screenshot to next `Code Reviewer` 🙏🏼 

![image](https://github.com/AnsarIbrahim/Budget-App/assets/63915024/ef69699c-a67b-4748-951c-a72e2632cd9d)